src/scripts/background.js

```diff
@@ -6,6 +6,7 @@ var currentTab;
 var isNeedDeleteTimeIntervalFromTabs = false;
 var activity = new Activity();
 var storage = new LocalStorage();
+var auth = new Auth();
```

Comment:
Would it make sense to call it kind of `AuthHelper` and make it static since it does not contains any state and usage of `this`

Reply:
I need it to be non-static to use it here
https://github.com/Bowery-RES/web-activity-time-tracker-whitelist/blob/feature/WEB-5507-add-sign-in-functionality/src/scripts/sign-in.js#L6

Reply:
Hmm, interesting, alright.